conf/vsc_ugent.config

```diff
@@ -0,0 +1,110 @@
+// Specify the work directory
+workDir = "$VSC_SCRATCH_VO_USER/work"
+
+// Perform work directory cleanup when the run has succesfully completed
+cleanup = true
+
+// Reduce the job submit rate to about 10 per second, this way the server won't be bombarded with jobs
+executor {
+    submitRateLimit = '10 sec'
+}
+
+// Specify that singularity should be used and where the cache dir will be for the images
+singularity {
+    enabled = true
+    autoMounts = true
+    cacheDir = "$VSC_SCRATCH_VO_USER/singularity"
+}
+
+// Define profiles for each cluster
+profiles {
+    skitty {
+        params {
+            config_profile_description = 'HPC_SKITTY profile for use on the Skitty cluster of the VSC HPC.'
+            config_profile_contact = 'Nicolas Vannieuwkerke (@nvnieuwk)'
+            config_profile_url = 'https://www.ugent.be/hpc/en'
+            max_memory = 177.GB
```

Comment:
Add the `module purge && module load` for each cluster to the relevant config?
This way, the user doesn't have to set those manually

Reply:
Is it possible for the jobs to change clusters? Because you will execute nextflow on the `victini` cluster and then change the cluster using `module load` or `module swap`.

Reply:
Are those commands on a per job base or per pipeline? 
If it's per job, then you're right, it doesn't make sense.

Reply:
It's per job :)

Reply:
nvm then!